src/Parser/Druid/Feral/Modules/Features/EnergyCapTracker.js

```diff
@@ -0,0 +1,138 @@
+ *
+ * No need to override getReducedDrain:
+ * Reduced drain cost from Berserk/Incarnation on Ferocious Bite is already applied in the log.
+ * Free bites from T21_4pc don't generate a drain event in the log.
+ */
+class EnergyCapTracker extends RegenResourceCapTracker {
+  static resourceType = RESOURCE_TYPES.ENERGY;
+  static baseRegenRate = BASE_ENERGY_REGEN;
+  static isRegenHasted = true;
+  static cumulativeEventWindow = 400;
+  static buffsChangeMax = [
+    SPELLS.BERSERK.id,
+    SPELLS.INCARNATION_KING_OF_THE_JUNGLE_TALENT.id,
+  ];
+  static resourceRefundOnMiss = RESOURCE_REFUND_ON_MISS;
+  static exemptFromRefund = [
+    SPELLS.THRASH_FERAL.id,
+    SPELLS.CAT_SWIPE.id,
+    SPELLS.BRUTAL_SLASH_TALENT.id,
+  ];
+
+  getReducedCost(event) {
+    let cost = super.getReducedCost(event);
+    if (!cost) {
+      return 0;
+    }
+    // no need to check for Clearcasting as the zero cost is already applied in the log
+    // no need to check for T21_4pc as the free bite already shows as free in the log
+    
+    if (this.selectedCombatant.hasBuff(SPELLS.BERSERK.id) ||
+        this.selectedCombatant.hasBuff(SPELLS.INCARNATION_KING_OF_THE_JUNGLE_TALENT.id)) {
+      cost *= BERSERK_COST_MULTIPLIER;
+      debug && console.log(`Cost reduced to ${cost} by Berserk/Incarnation`);
+    }
+
+    return cost;
+  }
+
+  naturalRegenRate() {
+    let regen = super.naturalRegenRate();
+    if (!debugIsPlayerAbove115 && this.selectedCombatant.getItem(ITEMS.CHATOYANT_SIGNET.id)) {
+      // BFA: Chatoyant's effect will stop working after level 115.
+      regen *= CHATOYANT_SIGNET_REGEN_MULTIPLIER;
+    }
+    return regen;
+  }
+
+  currentMaxResource() {
+    let max = BASE_ENERGY_MAX;
+    if (!debugIsPlayerAbove115 && this.selectedCombatant.getItem(ITEMS.CHATOYANT_SIGNET.id)) {
+      // BFA: Chatoyant's effect will stop working after level 115.
+      max += CHATOYANT_SIGNET_MAX_ADDITION;
+    }
+    if (this.selectedCombatant.hasTalent(SPELLS.MOMENT_OF_CLARITY_TALENT_FERAL.id)) {
+      max += MOMENT_OF_CLARITY_MAX_ADDITION;
+    }
+    if (this.combatantHasBuffActive(SPELLS.BERSERK.id) || this.combatantHasBuffActive(SPELLS.INCARNATION_KING_OF_THE_JUNGLE_TALENT.id)) {
+      // combatantHasBuffActive is used so that if the buff faded at this timestamp it will not count.
+      max += BERSERK_MAX_ADDITION;
+    }
+    if (!debugIsPlayerAbove115 && this.selectedCombatant.getItem(ITEMS.STABILIZED_ENERGY_PENDANT.id)) {
+      // BFA: Pendant's effect will stop working after level 115.
+      // multiplier is applied after the additions
+      max *= STABILIZED_ENERGY_PENDANT_MAX_MULTIPLIER;
+    }
+    // What should be x.5 becomes x in-game.
+    return Math.floor(max);
+  }
+
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<Icon icon="spell_shadow_shadowworddominate" alt="Capped Energy"/>}
+        value={`${formatPercentage(this.cappedProportion)}%`}
+        label="Time with capped energy"
```

Comment:
How about “Watsed energy” instead? I think that would be easier to grasp, and match the goal of these analyzers even more.